### ISSUE TITLE
fine tune on pretrained KittiSeg weights

### ISSUE THREAD

**Issue by yanweiw** [MarvinTeichmann/KittiSeg#178]
Hi Marvin, could you talk about why you opt for fine tuning on the VGG16 weights rather than your KittiSeg weights? I have a binary segmentation dataset that's quite similar to your road segmentation data but with only 100 images. I was hoping to leverage tv-continue to fine tune on the fcn8 weights you provided. Can you give me some hints on how to do that?

Going off that, I was also hoping to reset the weights in final convolutional layers of fcn8 for better tuning towards my data domain. My ground truth labels are black/white binary masks, and I was wondering if I could avoid overlaying them on my original images to compile with the mask format required to train with your existing code? Could you provide some details of how you would go about it? Thanks a lot.
